ui/jobs/jobs.js

```diff
@@ -1493,7 +1499,7 @@ class Bars {
       // Because thresholds are nonmonotonic (when finishing a combo)
       // be careful about setting them in ways that are visually poor.
       if (eyeBox.value >= oldThreshold &&
-          eyeBox.value >= newThreshold)
```

Comment:
Why are you changing these? I feel like this and line 1502 are more readable like this.  If I could figure out how to make eslint enforce this indentation, I would turn on a rule.

Reply:
Oops, it might be changed by my auto-formatting tools.
I'll revert it back later.